### ISSUE TITLE
koin load moudle

### ISSUE THREAD

**Issue by Aiyuan-h** [InsertKoinIO/koin#1320]
**Describe the bug**
In version 2.0.1 of koin, use the loadKoinModules api to load 150 modules, which takes 100ms. Is the performance of the latest version of koin improved?

**To Reproduce**
fun loadModules(context: Context, modules: List<Module>) {
        BasicLog.info(TAG, "loadModules begin $context")
        if (!isInitialized()) {
            doStartKoin(context)
        }
        try {
            BasicLog.info(TAG, "loadKoinModules")
            loadKoinModules(modules)
        } catch (e: DefinitionOverrideException) {
            BasicLog.warn(TAG, "already load!")
        }
        BasicLog.info(TAG, "loadModules end")
    }

**Koin project used and used version (please complete the following information):**
 "org.koin:koin-android:2.0.1"

**Comment by egorikftp** [comment]
Why don't you try out the latest version?

**Comment by arnaudgiuliani** [comment]
yes 2.0.1 is very old and many improvements have been done since then. Try upgrade to latest 3.1.x
